packages/app/src/app/components/Markdown/Link.tsx

```diff
@@ -1,12 +1,22 @@
-import { Button, Link } from '@codesandbox/components';
+import { Button, Link, Text } from '@codesandbox/components';
 import css from '@styled-system/css';
 import { useOvermind } from 'app/overmind';
 import React from 'react';
 
 export const LinkElement = ({ href, children, ...props }) => {
+  let commentId = '';
```

Comment:
Really small nit: this way `commentId` can be only two things:

1. null
2. a valid comment id

instead of 3 things:

1. null
2. valid comment id
3. empty string

```suggestion
  let commentId = null;
```

Reply:
Good catch, changed 👍